src/main/kotlin/org/jitsi/nlj/transform/node/incoming/RtxHandler.kt

```diff
@@ -50,8 +50,24 @@ class RtxHandler : TransformerNode("RTX handler") {
      */
     private val associatedSsrcs: ConcurrentHashMap<Long, Long> = ConcurrentHashMap()
 
-    companion object {
-        val logger: Logger = Logger.getLogger(this::class.java)
+    init {
+        streamInformationStore.onRtpPayloadTypesChanged { currentPayloadTypes ->
```

Comment:
i think overall it's still a net win, but i do think some things like this get a bit more awkward.  i didn't want to have to keep track of 2 different handlers for 2 different events for payload types (payload type added AND payload types cleared--this is similar to the old code but the verbosity of it felt a bit more apparent with having to track 2 lists of handlers in `StreamInformationStore`, plus this method has the advantage that a subscriber can't 'forget' to subscribe to/handle the 'clear' case) so i did a single event with the current state.  the downside of this is that we've got to iterate over the whole current state again.  it's not bad as this isn't going to happen often and isn't in a critical path, but i don't think it looks quite as nice.